cmd/prom2json/main.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2014 Prometheus Team
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"runtime"
+
+	"github.com/prometheus/log"
+
+	dto "github.com/prometheus/client_model/go"
+	"github.com/ChristianKniep/prom2json"
+)
+
+func main() {
+	runtime.GOMAXPROCS(2)
```

Comment:
This was 5 before. Why change it?

In different news, with modern Go versions, we don't even need this anymore. I'd just remove the line.